src/base/bittorrent/session.cpp

```diff
@@ -1792,6 +1792,14 @@ void Session::processShareLimits()
                             logger->addMessage(tr("'%1' reached the maximum ratio you set. Removed.").arg(torrent->name()));
                             deleteTorrent(torrent->hash());
                         }
+                        else if (m_maxRatioAction == Delete) {
+                            logger->addMessage(tr("'%1' reached the maximum ratio you set. Deleted.").arg(torrent->name()));
```

Comment:
Maybe instead of `Deleted` use 
Removed torrent + files
Removed torrent + files (force)
To be more in line with the combobox and clarity.